beacon.go

```diff
@@ -30,25 +31,40 @@ import (
 )
 
 type beaconConfig struct {
-	projectID string
-	tableId   string
+	ApiVersion string
+	ProjectID  string
+	TableId    string
 }
 
 const (
-	projectKey = "GOOGLE_CLOUD_PROJECT"
-	bqTableKey = "GOOGLE_BIGQUERY_TABLE"
+	apiVersionKey = "VERSION"
+	projectKey    = "GOOGLE_CLOUD_PROJECT"
+	bqTableKey    = "GOOGLE_BIGQUERY_TABLE"
 )
 
-var config = beaconConfig{
-	projectID: os.Getenv(projectKey),
-	tableId:   os.Getenv(bqTableKey),
-}
+var (
+	aboutTemplate = template.Must(template.ParseFiles("about.xml"))
```

Comment:
Do you need to list this in the app.yaml at all or it automatically picks it up?

Reply:
because I have the handler for url: /.* it picks up all paths. If I go to anything else except /query it will show the beacon info, so it won't display an error for /wrong. If I want an error I should do 
- url: /
  script: _go_app
- url: /query
  script: _go_app